src/plugins/intel_cpu/src/emitters/snippets/x64/utils.hpp

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2023 Intel Corporation
```

Comment:
```suggestion
// Copyright (C) 2023-2024 Intel Corporation
```

Reply:
It probably should be just 2024, since I've just created the file